modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -87,6 +87,10 @@ By default, database metrics include:
 |<prefix>.bolt.messages_failed|The total number of messages that have failed while processing. A high number of failures may indicate an issue with the server and further investigation of the logs is recommended. (counter)
 |<prefix>.bolt.accumulated_queue_time|(unsupported feature) When `internal.server.bolt.thread_pool_queue_size` is enabled,  the total time in milliseconds that a Bolt message waits in the processing queue before a Bolt worker thread becomes available to process it. Sharp increases in this value indicate that the server is running at capacity. If `internal.server.bolt.thread_pool_queue_size` is disabled, the value should be `0`, meaning that messages are directly handed off to worker threads. (counter)
 |<prefix>.bolt.accumulated_processing_time|The total amount of time in milliseconds that worker threads have been processing messages. Useful for monitoring load via Bolt drivers in combination with other metrics. (counter)
+|<prefix>.bolt_driver.managed_transaction_function_calls|The total number of managed transaction function calls over the lifetime of the sever from all drivers. (counter)
+|<prefix>.bolt_driver.unmanaged_transaction_calls|The total number of unmanaged transaction function calls over the lifetime of the sever from all drivers. (counter)
+|<prefix>.bolt_driver.implicit_transaction_calls|The total number of implicit transaction function calls over the lifetime of the sever from all drivers. (counter)
+|<prefix>.bolt_driver.execute_calls|The total number of execute function calls over the lifetime of the sever from all drivers. (counter)
```

Comment:
```suggestion
|<prefix>.bolt_driver.execute_calls|The total number of execute function calls over the lifetime of the server from all drivers. (counter)
```

Reply:
The total number of driver-level execute transaction function calls executed on this server.

Reply:
it could be that server A accepted the metric but failed to start a transaction, so server B executes the transaction.